commit-status-publisher-server/src/main/java/jetbrains/buildServer/commitPublisher/github/GetJiraTickets.java

```diff
@@ -0,0 +1,50 @@
+package jetbrains.buildServer.commitPublisher.github;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import com.intellij.openapi.diagnostic.Logger;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class GetJiraTickets {
+  private static final Logger LOG = Logger.getInstance(GetJiraTickets.class.getName());
+  private static final String od = "od\\d*";
```

Comment:
this pattern will match all candidate that contain **od**, we need a pattern to match **od d+** or **od-d+**